doc/optimizers.md

```diff
@@ -1394,6 +1394,114 @@ double Optimize(arma::mat& X);
  * [Semidefinite programming on Wikipedia](https://en.wikipedia.org/wiki/Semidefinite_programming)
  * [Semidefinite programs](#semidefinite-programs) (includes example usage of `PrimalDualSolver`)
 
+## Quasi-Hyperbolic Momentum Update
```

Comment:
Should this be called `Quasi-Hyperbolic Momentum Update SGD (QHSGD)`?  The anchors will need to be updated.  (In fact the `qhsgd` anchor right now doesn't point anywhere.)